packages/compiler/src/ml_parser/parser.ts

```diff
@@ -351,9 +351,10 @@ class _TreeBuilder {
       const quoteToken = this._advance();
       end = quoteToken.sourceSpan.end;
     }
+    const keySpan = new ParseSourceSpan(attrName.sourceSpan.start, attrName.sourceSpan.end);
```

Comment:
May be move the source span creation here too (for better readability) and use `sourceSpan` below?
```suggestion
    const sourceSpan = new ParseSourceSpan(attrName.sourceSpan.start, end, attrName.sourceSpan.fullStart);
```

Reply:
Oops! Sorry I missed this comment before merging 👀

Reply:
Haha, no worries, it's a small nit anyway!